core/src/main/scala/org/scalafmt/config/Newlines.scala

```diff
@@ -63,7 +105,9 @@ case class Newlines(
     penalizeSingleSelectMultiArgList: Boolean = true,
     alwaysBeforeCurlyBraceLambdaParams: Boolean = false,
     alwaysBeforeTopLevelStatements: Boolean = false,
-    afterCurlyLambda: NewlineCurlyLambda = NewlineCurlyLambda.never
+    afterCurlyLambda: NewlineCurlyLambda = NewlineCurlyLambda.never,
+    afterImplicitKWInVerticalMultiline: Boolean = false,
+    beforeImplicitKWInVerticalMultiline: Boolean = false
```

Comment:
What is the motivation for this option? I feel like `afterImplicitKWInVerticalMultiline` is a lot more common.